docker-compose-solo.yml

```
@@ -0,0 +1,63 @@
+
+volumes:
+    postgresql_data:
+
+services:
+  database:
+    image: georchestra/database:24.0.x
```

If you only need a "vanilla"/classic PostgreSQL, you don't need to use "georchestra/database" which comes with pre-existing data about geOrchestra (multiple schemas)

```suggestion
    image: postgres:16
```